client/src/components/Console.tsx

```diff
@@ -95,8 +91,18 @@ export default function Console() {
       fetchDocuments();
       setRefreshNeeded(false);
     }
-  }, [selectedView, refreshNeeded]);
+  }, [refreshNeeded]);
+  useMemo(() => {
+    const entireMunicipalityDocs = tmpDocuments.filter(
+      (doc) => doc.doc_coordinates?.type === "EntireMunicipality"
```

Comment:
I have a record with Persian language, but using the filter as I attached the pictures, there is no results
<img width="1440" alt="Screen Shot 2024-11-25 at 22 40 06" src="https://github.com/user-attachments/assets/c58e5403-a6e4-47ff-85a4-64b333b98327">
<img width="1440" alt="Screen Shot 2024-11-25 at 22 40 23" src="https://github.com/user-attachments/assets/1b6cec99-5435-449b-a3fe-3f748b143c15">
<img width="1440" alt="Screen Shot 2024-11-25 at 22 40 36" src="https://github.com/user-attachments/assets/2b08198c-4883-4f68-a24f-f3669ced9b35">